apiserver/debuglog.go

```diff
@@ -197,38 +197,61 @@ type debugLogParams struct {
 }
 
 func readDebugLogParams(queryMap url.Values) (debugLogParams, error) {
```

Comment:
hi @jack-w-shaw , do you mind helping to refactor this readDebugLogParams  function accordingly to this implementation? Thanks!

https://github.com/juju/juju/pull/18674/files